prow/crier/reporters/slack/reporter.go

```diff
@@ -19,7 +19,7 @@ package slack
 import (
 	"bytes"
 	"fmt"
-	"text/template"
+	templatePkg "text/template"
```

Comment:
I thought package name should not contain capital letter

Reply:
It's fine to contain capital letter in the package alias I think, but generally it looks we should always use lower-case, so I changed the alias to `texttemplate`.